sources/web/datalab/static/light.css

```diff
@@ -52,13 +57,18 @@ div.btn-toolbar .btn.active {
 span.autosave_status {
   color: #ddd;
 }
+div.cell.selected {
+  background: linear-gradient(to right, #42A5F5 -40px, #42A5F5 5px, transparent 5px, transparent 100%);
+  border: solid 1px #ccc !important;
+  box-shadow: 0px 0px 6px 0px rgba(0, 0, 0, 0.3);
+}
 div.code_cell.session.completed div.input {
   border-left: solid 4px #337ab7;
 }
 div.code_cell div.input {
   background-color: #f7f7f7;
 }
-.CodeMirror-selected {
+.codemirror-selected {
```

Comment:
Just noticed this ... its .CodeMirror in other rules; just double checking this was intended (I believe these things are case-sensitive).